### DISCUSSION TITLE
Community RFC Call - March 8, 2021

### DISCUSSION THREAD

**Discussion by FredKSchott** [withastro/roadmap#137] [Announcements]
# To Discuss

- None yet

_If there is something that you would like to see discussed, please comment below. We will prioritize pending RFC, but if we have time to spare we will open the call up to get attention on and discuss [other threads and feature requests.](https://github.com/withastro/rfcs/discussions)._

**Comment by bholmesdev** [comment]
Extracted an RFC comment to a fully-fledged RFC here #135. I think it relates to a few other RFCs being considered!

**Comment by FredKSchott** [comment]
Update: We're experimenting with an async format today, see Discord #general channel for more!

**Reply by sarah11918** [reply]
"async" as in, "Discord is down" async?  https://discordstatus.com/

Not sure whether everyone's affected, but I am!

**Reply by FredKSchott** [reply]
I was too! Back up now!
